R/pkg/R/column.R

```diff
@@ -132,17 +132,24 @@ createMethods()
 
 #' alias
 #'
-#' Set a new name for a column
+#' Set a new name for an object. Equivalent to SQL "AS" keyword.
```

Comment:
Also, I think this doc block (description, param list specifically) should be move to DataFrame.R or generic.R as mentioned before.